R-package/R/saveRDS.lgb.Booster.R

```diff
@@ -18,7 +18,7 @@
 #' @return NULL invisibly.
 #'
 #' @examples
-#' \dontrun{
+#' \donttest{
```

Comment:
Will `donttest` (actually all our examples) be run at RTD?
https://github.com/microsoft/LightGBM/blob/967a7814a7ca7b06e3eab119a88ae8206637e847/docs/conf.py#L261-L270

Reply:
There is no option in `?pkgdown::build_site` to explicitly enable `\donttest` examples, so I think they're run.

![image](https://user-images.githubusercontent.com/7608904/95288249-771ef480-082d-11eb-9289-10c3fd10cb81.png)

Reply:
I just added an error to an example wrapped in `\donttest{}`, then re-generated the `{pkgdown}` site locally and I can see that it was run!

![image](https://user-images.githubusercontent.com/7608904/95289648-eba76280-0830-11eb-8198-4907bcfb6ed4.png)

Reply:
OK, I see! Thanks a lot for your investigation!